docs/ref/contrib/admin/admindocs.txt

```diff
@@ -12,14 +12,18 @@ docstrings of models, views, template tags, and template filters for any app in
 :setting:`INSTALLED_APPS` and makes that documentation available from the
 :mod:`Django admin <django.contrib.admin>`.
 
-You may, to some extent, utilize :mod:`~django.contrib.admindocs` to quickly
-document your own code. This has limited usage, however, as the app is
-primarily intended for documenting templates, template tags, and filters.
-For example, model methods that require arguments are purposefully omitted
-from the documentation because they can't be invoked from templates. The app
-can still be useful since it doesn't require you to write any extra
-documentation (besides docstrings) and is conveniently available from the
-:mod:`Django admin <django.contrib.admin>`.
+This way, Django admin documentation generator provides offline documentation
+for all custom and built-in template tags and template filters.
+
+.. admonition:: Note
```

Comment:
I don't see why this was moved to a note box (by the way, there is `.. note::`), but anyway, you should have a ``.. versionchanged:: 1.9` note as well as a mention in the release notes. See our [patch review checklist](https://docs.djangoproject.com/en/dev/internals/contributing/writing-code/submitting-patches/#patch-review-checklist).